internal/command/extensions/sentry/sentry.go

```diff
@@ -17,3 +17,116 @@ func New() (cmd *cobra.Command) {
 
 	return cmd
 }
+
+var Platforms = []string{
+	"android",
+	"apple-ios",
+	"apple-macos",
+	"capacitor",
+	"cocoa-objc",
+	"cocoa-swift",
+	"cordova",
+	"csharp",
+	"dart",
+	"dart-flutter",
+	"dotnet",
+	"dotnet-aspnet",
+	"dotnet-aspnetcore",
+	"dotnet-awslambda",
+	"dotnet-gcpfunctions",
+	"dotnet-maui",
+	"dotnet-winforms",
+	"dotnet-wpf",
+	"dotnet-xamarin",
+	"electron",
+	"elixir",
+	"flutter",
+	"go",
+	"go-http",
+	"ionic",
+	"java",
+	"java-android",
+	"java-appengine",
+	"java-log4j",
+	"java-log4j2",
+	"java-logback",
+	"java-logging",
+	"java-spring",
+	"java-spring-boot",
+	"javascript",
+	"javascript-angular",
+	"javascript-angularjs",
+	"javascript-backbone",
+	"javascript-capacitor",
+	"javascript-cordova",
+	"javascript-electron",
+	"javascript-ember",
+	"javascript-gatsby",
+	"javascript-nextjs",
+	"javascript-react",
+	"javascript-remix",
+	"javascript-svelte",
+	"javascript-vue",
+	"kotlin",
+	"minidump",
+	"native",
+	"native-breakpad",
+	"native-crashpad",
+	"native-minidump",
+	"native-qt",
+	"node",
+	"node-awslambda",
+	"node-azurefunctions",
+	"node-connect",
+	"node-express",
+	"node-gcpfunctions",
+	"node-koa",
+	"perl",
+	"php",
+	"php-laravel",
+	"php-monolog",
+	"php-symfony2",
+	"python",
+	"python-awslambda",
+	"python-azurefunctions",
+	"python-bottle",
+	"python-celery",
+	"python-django",
+	"python-fastapi",
+	"python-flask",
+	"python-gcpfunctions",
+	"python-pylons",
+	"python-pyramid",
+	"python-rq",
+	"python-sanic",
+	"python-starlette",
+	"python-tornado",
+	"react-native",
+	"ruby",
+	"ruby-rack",
+	"ruby-rails",
+	"rust",
+	"unity",
+	"unreal",
+}
+
+var PlatformMap = map[string]string{
+	"Bun":           "javascript",
```

Comment:
This is just a question.  Deno does not have as a goal to be node-compatible, though recently it has done some effort along these lines.  Bun does have a goal to be node-compatible... it isn't perfect, but is steadily improving along those lines.  With this context in mind, why does Deno map to node and Bun map to javascript?

Reply:
That was a bit arbitrary. I don't know that there's a big difference in how Sentry handles node versus javascript projects. We can switch it to node.

Reply:
I added a comment with a link to the list of supported platforms.